android/src/main/kotlin/com/appcues/flutter/FlutterElementTargetingStrategy.kt

```diff
@@ -81,11 +81,19 @@ internal class FlutterElementTargetingStrategy(val plugin: AppcuesFlutterPlugin)
             val displayMetrics = it.context.resources.displayMetrics
             val density = displayMetrics.density
 
+            // get the root level window insets, which will be used to apply a height correction below
+            val insets = ViewCompat.getRootWindowInsets(it)?.getInsets(WindowInsetsCompat.Type.systemBars())
+                ?: Insets.NONE
+
+            // the capture from the FlutterRenderer contains the top system bar but not the bottom
+            // so bottom is subtracted from the layout height
+            val height = (actualPosition.height() - insets.bottom).toDp(density)
```

Comment:
this is what fixed the selector drawing issue - as the incorrect (too tall) height was causing the `scale` value used here to be incorrect https://github.com/appcues/appcues-android-sdk/blob/main/appcues/src/main/java/com/appcues/debugger/ui/CaptureConfirmDialog.kt#L143

another option would be to update the native SDK to instead use `capture.metadata.deviceHeight.dp` - which is really the image size value and maybe slightly more correct, but in reality - they should both match and be usable. So, making the fix here to start and keep it simple (no native change)